commands/google.py

```diff
@@ -0,0 +1,36 @@
+# GOOGLE COMMAND NOT IN USE
```

Comment:
y tho

Reply:
Because I didn't want to make it work. it has weird bugs that I'll need to work out in the future.

Reply:
+1